libs/genai/langchain_google_genai/_function_utils.py

```diff
@@ -109,6 +109,17 @@ def _format_json_schema_to_gapic(schema: Dict[str, Any]) -> Dict[str, Any]:
             continue
         elif key in ["type", "_type"]:
             converted_schema["type"] = str(value).upper()
+        elif key == "allOf":
```

Comment:
It looks to me this implementation is a little bit cleaner:
https://github.com/langchain-ai/langchain-google/blob/e0c17f07bda807cdad97e9f5523a3093cca6f579/libs/vertexai/langchain_google_vertexai/functions_utils.py#L95

wdyt? Do you mind taking the same approach, please?